app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -139,6 +160,12 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
     props.projectRevision
   );
 
+  const fundingStream =
+    projectRevision.projectFormChange.asProject
+      .fundingStreamRfpByFundingStreamRfpId.fundingStreamByFundingStreamId.name;
+
+  const isFundingStreamEP = fundingStream === "EP";
```

Comment:
It could be good to verify that the funding stream is "IA" and not just some erroneous value (see `formStructure.tsx`). This could also catch errors later if another stream is added

Reply:
If we are concerned about it, an alternative is to have a check that it's either IA or EP, then keep the single `isFundingStreamEP` variable

Reply:
I think I'm not concerned about erroneous funding stream values because the validation plugin and db foreign key errors would have gone off before this point (but feel free to disagree strongly and I can change). 

If we get any other funding streams in the future, this can be changed then